app/pocket.go

```diff
@@ -56,11 +58,16 @@ func NewPocketBaseApp(logger log.Logger, db db.DB, options ...func(*bam.BaseApp)
 	tkeys := sdk.NewTransientStoreKeys(nodesTypes.TStoreKey, appsTypes.TStoreKey, pocketTypes.TStoreKey, gov.TStoreKey)
 	// add params Keys too
 	// Create the application
+
+	// setup a global cache system
+	globalCache := sdk.NewCache(10)
```

Comment:
Need configuration

Reply:
added a flag @ CLI level to allow node-runners to increase and decrease cache size